api/src/gameState/gameState.service.ts

```diff
@@ -399,6 +421,16 @@ export class GameStateService {
     );
     if (itWillExitAnOwnMarble) {
       throw new CantMoveError("You can't exit one of your own marbles");
+    } 
+
+    const marbleCoordinate = marblePosition.split(',');
```

Comment:
It's generally a good idea to move this logic in a specific method. (convert the string into coordinates)